keras_cv/datasets/waymo/load.py

```diff
@@ -43,6 +43,11 @@ def load(
     """
     Loads the Waymo Open Dataset and transforms frames into features as
     tensors.
+
+    References:
+        - Waymo Dataset Research Paper: https://arxiv.org/abs/1912.04838
```

Comment:
Let's replace this style with `[Waymo Dataset Research Paper](https://arxiv.org/abs/1912.04838)`. Thanks!